serde_v8/Cargo.toml

```diff
@@ -12,12 +12,18 @@ description = "Rust to V8 serialization and deserialization"
 [lib]
 path = "lib.rs"
 
+[features]
+default = []
+# Requires a recent Linux kernel version (atleast )
```

Comment:
```suggestion
# Requires a recent Linux kernel version (atleast 5.6; tested on >5.11.0)
```

I think 5.6 is safe to assume? Read op was added in 5.6. I'm on 5.15.0 and tokio_iouring seems to suggest >5.11.0

Reply:
IORING_OP_READV and IORING_OP_READ_FIXED - what maps to the Readv and ProvideBuffers ops, if I read the the io_uring crate's source code correctly - were added in 5.1.